### DISCUSSION TITLE
Modeling of Proteins

### DISCUSSION THREAD

**Discussion by Salih-Benamara** [Open-Systems-Pharmacology/Forum#1170] [Questions & Problems]
Hello,

Please, in the model published by Malik and Edginton - 2019, the expression of TNF (target of infliximab) for the reaction that manages the synthesis of antigen (TNF), I found this parameter "Relative expression out" but I can not find how to obtain or create it ( in Pk-SIM or Mobi) for other models. I would like to know how it can be expressed in the software and what it is used for (difference with the normal expression).

https://ascpt.onlinelibrary.wiley.com/doi/full/10.1002/psp4.12456 .
The model support file is in the Supporting information (Projet File) in the following article
The parameter can be found in ''Reactions''>TNF_Synthesis>Properties (building blocks)> Relative expression out.
or in '' Parameter Start Values " > Parameter Name>  Relative expression out.
![question](https://user-images.githubusercontent.com/100346781/158190136-716ca2d7-5ab1-490e-acd4-24dd9fa00045.JPG)
![Question 2](https://user-images.githubusercontent.com/100346781/158190168-f67445ac-677f-41d6-b663-ed4735b33376.JPG)
 
Thank you.

**Comment by prvmalik** [comment]
Hi, it will not be able to be run in versions beyond 9.1. Impossible. The approach to interstitial target expression in organs is changed.

**Reply by Salih-Benamara** [reply]
Thank you very much for the reply, so we can not run THE TMDD on the new versions ... /:

**Reply by msevestre** [reply]
>it will not be able to be run in versions beyond 9.1. Impossible.

>They did not know it was impossible so they did it

So maybe someone will make it work?

**Reply by Salih-Benamara** [reply]
any suggestions for running a TMDD model on the V10 please ....

**Reply by Salih-Benamara** [reply]
@msevestre Hi, please any suggestions on how can i run this reactions ?? AND EXPRESSION ?? Thank you very much  :D :D on V10 for Infliximab and TNF

**Reply by prvmalik** [reply]
No, TMDD can be run on new versions. However models built on 9.1 will need to be re-estimated/re-calibrated for version 10. Target expression is changed between V9.1 and V10.

**Comment by prvmalik** [comment]
Hi, you can re-estimate the model for V10, or create a new model with the infliximab data for V10. PKSim is not backward compatible with previous versions.

**Reply by Salih-Benamara** [reply]
I create a new model with the infliximab data for V10 on PK-Sim (V10) then i export the Project on Mobi (V10) but for the synthesis reaction there is always the same error with the path ...whene i run the simulation

**Reply by prvmalik** [reply]
That issue is being discussed on thread (1151) which you have just chimed in on.

As a word of advice infliximab is a terribly complex molecule due to the high immunogenicity. If re-estimating I would advise maybe restricting to single dose administration so that immunogenicity is not an issue, and removing that component. Depends on the purpose of your model though. If you want to simulate over >3 months in patients you would need to account for immunogenicity in some fashion.

**Reply by Salih-Benamara** [reply]
I really appreciate your answer and thank you for your time, one last question if you don't mind please, in the reaction tabs, on V10 how can i run the model with those reaction ?? ( if i want to run it for another molecule without immunogenicity problems for exemple) i change just '' Relative Expression out.'' with '' Relative Expression '' is that correct ?? 

i have the same problem with IL6R 
![questionnn 2](https://user-images.githubusercontent.com/100346781/158209643-b56504d6-0e8f-4470-aa85-ee42ae177497.JPG)

**Reply by prvmalik** [reply]
I don't know because I have not yet learned that part of V10, but I assume that would be a path to pursue.
